.github/actions/verify-compose/action.yml

```diff
@@ -0,0 +1,54 @@
+name: "Verify and test docker compose"
```

Comment:
if all the steps of this actions make a workflow, just declare a reusable workflow instead of an action https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Reply:
well in the original workflow in acs-deployment this makes up just one job out of four, and that workflow differs from the aps-deployment so I don't think making it a workflow will be beneficial for our current usecase.